utils/src/config.rs

```diff
@@ -136,25 +119,24 @@ pub struct ServerTimeout {
     #[serde(with = "duration_format", default = "default_rpc_timeout")]
     rpc_timeout: Duration,
 
-    /// How long a candidate should wait before it starts another round of election
+    /// How many rounds of heartbeat_interval a candidate needs to wait until it starts a new round of election
+    /// It should be larger than `follower_timeout`
     #[getset(get = "pub")]
-    #[serde(with = "duration_format", default = "default_candidate_timeout")]
-    candidate_timeout: Duration,
+    #[serde(default = "default_candidate_timeout")]
+    candidate_timeout: u8,
```

Comment:
timeout should be a `Duration`, I think it should be renamed.

Reply:
No it's not real timeout duration but a multiplier. For example, real candidate timeout = candidate_timeout * heartbeat_interval. See the comment above.

Reply:
I mean this variable should not be called `*_timeout`, because other timeouts are `Duration`, maybe call it `candidate_round`?

Reply:
Ok. I will change it